app/config/set_parameters.php

```diff
@@ -57,10 +57,13 @@
 
     $driver = 'array';
     $cacheType = [
-        'CacheMemcache' => ['memcache'],
         'CacheMemcached' => ['memcached'],
-        'CacheApc' => ['apcu', 'apc'],
-        'CacheXcache' => ['xcache'],
+        'CacheApc' => ['apcu'],
```

Comment:
To be consistent since we remove these two solutions we need to:
- clean them from the BO form https://github.com/PrestaShop/PrestaShop/blob/3d06748975e00c548e940bf82862d9d211ae94d4/src/PrestaShopBundle/Form/Admin/AdvancedParameters/Performance/CachingType.php#L57
- change the default value here https://github.com/PrestaShop/PrestaShop/blob/ead6f332956495b2b53c84a3ef8e1fc6169c98e1/src/PrestaShopBundle/Install/Upgrade.php#L1206
- change the value of the const `_LEGACY_PS_CACHING_SYSTEM_` (I don't know how we can force the change of a const value, but since it's editable via the BO I guess there's a way)
- when the const is not used seems like the value is stored in `parameters.php` but it can be edited thanks to `prestashop.adapter.php_parameters`
- see if there are other places where we need to clean something

Reply:
Actually, apc & xcache are still available in PrestaShop's own implementation of the cache.